core/templates/dev/head/domain/statistics/FeedbackImprovementCardObjectFactorySpec.ts

```diff
@@ -150,15 +150,15 @@ describe('FeedbackImprovementCardObjectFactory', function() {
         FeedbackImprovementCardObjectFactory.createNew(this.mockThread);
     });
 
-    describe('.isOpen', function() {
-      it('returns true when status is open', function() {
+    describe('.getStatus', function() {
+      it('returns open when status is open', function() {
```

Comment:
Seems like all this should just change to a test which checks that "it returns open when it's open, it returns fixed when it's fixed, etc." -- not sure why "open" is special-cased in the new generic "get status" framing.

Reply:
Done.